webnode/src/redux/services/iota.js

```diff
@@ -39,26 +28,44 @@ const parseMessage = message => {
   return iota.utils.fromTrytes(evenChars);
 };
 
-const findMostRecentTransaction = address =>
+const findAllTransactions = address =>
   new Promise((resolve, reject) => {
     iotaProvider.api.findTransactionObjects(
       { addresses: [address] },
       (error, transactionObjects) => {
-        if (error || !transactionObjects.length) {
-          console.log("IOTA ERROR: ", error);
-          reject(new Error("No transaction found"));
-        }
         const settledTransactions = transactionObjects || [];
-        const recentTransaction = _.maxBy(
+        const ordered = _.orderBy(
           settledTransactions,
-          "attachmentTimestamp"
+          ["attachmentTimestamp"],
+          ["desc"]
         );
-        console.log("IOTA TRANSACTION: ", recentTransaction);
-        resolve(recentTransaction);
+
+        console.log("IOTA TRANSACTIONS: ", ordered);
+        resolve(ordered);
       }
     );
   });
 
+const checkIfClaimed = address =>
+  findAllTransactions(address).then(transactions => {
+    if (!transactions.length) return false;
+
+    const mostRecentTransaction = transactions[0];
+
+    const workedOnByOtherWebnode = transactions.length > 1;
+
+    const attachedAt = mostRecentTransaction.attachmentTimestamp;
+    const lastEpoch = moment()
+      .subtract(1, "minute")
```

Comment:
reattach transaction if 1+ minute old in the meantime while we are still debugging